rebasehelper/spec_hooks/pypi_url_fix.py

```diff
@@ -34,7 +34,7 @@ class PyPIURLFixHook(BaseSpecHook):
     NAME = 'PyPI URL Fix'
     CATEGORIES = ['python']
     SOURCES_URL_TRANSFORMATIONS = [
-        ('https?://pypi.python.org/', 'https://files.pythonhosted.org/'),
+        (r'https?://pypi(.python)?.org/', 'https://files.pythonhosted.org/'),
```

Comment:
This is almost fine, only `.` matches any character, not just literal `.`, so it would be safer to escape it.